src/packaging/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Jit": "2.0.0-beta-25117-06",
+    "Microsoft.NETCore.Jit": "2.0.0-preview1-25204-02",
```

Comment:
I also had to add ```"osx.10.12-x64": { },``` to the list of runtimes in this file to make it work on my computer.